test/dom.test.js

```diff
@@ -1,5 +1,38 @@
 import { h, app } from "../src"
 
+function testTreeSegue(name, trees) {
```

Comment:
@okwolf What's happening here?

Reply:
@JorgeBucaran I'm not familiar with this function, all I did was move it from the bottom of the file up toward to top.